pegasus/sites.v3/code.org/public/dance-landing.haml

```diff
@@ -0,0 +1,33 @@
+%h1= I18n.t(:dance_main_heading)
+
+.dance-subheading
+  %h4= I18n.t(:dance_subheading)
+
+%link{href: "/css/starwars.css", rel: "stylesheet"}
+%link{href: "/css/dance-landing.css", rel: "stylesheet"}
+
+%div{style: "clear:both"}
+
+.dance-container
+  .img-container
+  %h1.dance-heading= I18n.t(:starwars_subtitle)
+
+  .col-25.dance-tutorial.dance-js-tutorial
+    .tutorial-box
+      %h2.dance-h2= I18n.t(:dance_party_title)
+      .dance-tutorial-info
+        %p.dance-tutorial-description= I18n.t(:dance_party_description)
+        %p.dance-tutorial-specs= I18n.t(:dance_party_specs)
+      %a{:href => '/api/hour/begin/starwars', :target=>'_self'}
+        %button.dance-try-button= I18n.t(:try_now)
+
+  .col-25.dance-tutorial.dance-blocks-tutorial
+    .tutorial-box
+      %h2.dance-h2 Try Another Hour
```

Comment:
i18n?

Reply:
The title of the block used in the mockup increased the size of the block.  This is a potential fix for issue.  Awaiting confirmation from product team.